src/utils.ts

```diff
@@ -252,6 +279,14 @@ const getStoryIcon = (storyType: string): string => {
   }
 };
 
+const getPivotalReviewReminders = (reviews: Review[], memberships: PivotalProjectMembership[]): string[] => {
+  return reviews.map((review) => {
+    const member = memberships.find(member => member.person.id && member.person.id === review.reviewer_id);
+
+    return `- [ ] ${review.review_type.name} ${member && member.person && member.person.name ? `(${member.person.name})`: ''}`;
```

Comment:
Would it be worth doing this?

```suggestion
    return `- [ ] ${review.review_type.name} ${member && member.person && member.person.name ? `(${member.person.name})`: 'Unknown'}`;
```

What's the case where the name isn't known? Does that happen if no reviewer is set? Or are there other cases?

Reply:
if there isn't any person assigned to the review, it would just be [ ] Designer. The code should already cover that